src/pipeline_v3/utils/config.py

```
@@ -132,6 +163,7 @@ class PipelineConfig:
     chunking: ChunkingSettings = field(default_factory=ChunkingSettings)
     storage: StorageSettings = field(default_factory=StorageSettings)
     search: SearchSettings = field(default_factory=SearchSettings)  # NEW
+    processing_profiles: ProcessingProfileSettings = field(default_factory=ProcessingProfileSettings)  # NEW
```

Pylint (3.3.7)

[convention] Line too long (108/100)

(C0301)